src/components/Property/Contacts/TenantsContactTable/modals/SetAsMainModal.js

```diff
@@ -0,0 +1,79 @@
+import { frontEndApiRequest } from '@/root/src/utils/frontEndApiClient/requests'
+import ConfirmationModal from '../../../../ConfirmationModal'
+import { useState, useEffect } from 'react'
+
+const SetAsMainModal = ({
+  showModal,
+  setShowModal,
+  phoneNumber,
+  tenant,
+  reloadContacts,
+}) => {
+  const [isLoading, setIsLoading] = useState(false)
+  const [modalError, setModalError] = useState(null)
+
+  useEffect(() => {
+    // Reset on reopen modal
+    setModalError(null)
+  }, [showModal])
+
+  const handleOnSubmit = () => {
+    setIsLoading(true)
+    setModalError(null)
+
+    const body = {
+      contactInformation: {
+        contactType: phoneNumber.contactType,
+        value: phoneNumber.value,
+        description: phoneNumber.description,
+        subType: 'mainNumber',
+        addressExtended: phoneNumber?.addressExtended,
+      },
+    }
+
+    const path = `/api/contact-details/?contactId=${phoneNumber.contactId}&personId=${tenant.personId}`
+
+    frontEndApiRequest({
+      method: 'PATCH',
+      path,
+      requestData: body,
+    })
+      .then((res) => {
+        console.log({ res })
+
+        setShowModal(false)
+      })
+      .catch((err) => {
+        console.log({ err })
+        setModalError(err.message)
+      })
+      .finally(() => {
+        setIsLoading(false)
+        reloadContacts()
+      })
+  }
+
+  return (
+    <ConfirmationModal
+      title={'Set as main contact'}
+      showDialog={showModal}
+      setShowDialog={setShowModal}
+      isLoading={isLoading}
+      modalError={modalError}
+      modalText={
+        <p className="govuk-body">
+          Are you sure you want to change the contact type from{' '}
+          <strong style={{ textTransform: 'capitalize' }}>
```

Comment:
This may be a bit easier to read with these <strong> subsections set as variables before the return

Reply:
Im not sure what you mean. Could you provide an example?

Reply:
I was thinking of moving them to mini components and just doing:
```js
<p className="govuk-body">
  Are you sure you want to change the contact type from
  <PhoneNumberSubType /> to <MainNumber />?
</p>
```
but the components for this are too large so it doesn't actually add clarity